### PR TITLE
Added a description for the seq command.

### LINKED ISSUE #185: seq
Claiming seq

### PR DESCRIPTION
**Pull request by tlaredo**
# Advanced HW 9 submission

Closes #185 

Name: Tyler Laredo
Uniqname: tlaredo

### CODE REVIEW

**Comment by mterwill** [comment]
Cool; didn't know about this one. Seems useful!

File: _commands/basics/seq.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+~~~
+
+<!--more-->
+
+where `seq` prints numbers from *FIRST* to *LAST* by an increment *STEP*.
+Default: *FIRST* & *STEP* are both 1. Each number is printed on its own line. Seq can be used on all real numbers.
+
+### Useful Options / Examples
+
+#### `-f, --format=`*FORMAT*`
```
Comment (mterwill) [inline _commands/basics/seq.md]: This is rendering with an extra backtick 
screen shot 2016-03-25 at 11 06 35 pm

File: _commands/basics/seq.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+---
+
+seq
+-------
+### Overview
```
Comment (mterwill) [inline _commands/basics/seq.md]: You can delete this (it shows up on the homepage and looks weird)
image

File: _commands/basics/seq.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+EXAMPLE:
+
+~~~ bash
+$ seq -f "tyler%02" 10
+~~~
+
+will produce:
+
+tyler01
```
Comment (mterwill) [inline _commands/basics/seq.md]: Newlines don't show up, I think it's a Markdown thing. There's probably a way to force them, though.

image

**Comment by tlaredo** [comment]
Thanks for the suggestions! I just made the changes and pushed them to my master branch. How do I implement these changes? Do I have to issue a new pull request?

File: _commands/basics/seq.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+---
+
+seq
+-------
+
+`seq`prints a sequence of numbers to standard ouput.
+
+~~~ bash
```
Comment (ppannuto) [inline _commands/basics/seq.md]: Could you insert the output of the simple, correct operation case here, i.e.:

```
$ seq 3
1
2
3
```

Right before the detail line you have here?

File: _commands/basics/seq.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+<!--more-->
+
+where `seq` prints numbers from *FIRST* to *LAST* by an increment *STEP*.
+Default: *FIRST* & *STEP* are both 1. Each number is printed on its own line. Seq can be used on all real numbers.
+
+### Useful Options / Examples
+
+#### `-f, --format=`*FORMAT*
+
+prints all numbers using *FORMAT*, where the default is `%g`. *FORMAT* must contain one of the following float output formats: `%g`, `%e`, `%f`.
```
Comment (ppannuto) [inline _commands/basics/seq.md]: I'm a little confused, you say that FORMAT has to have one of %g, %e, or %f, but your example below this doesn't have any of those?

**Comment by ppannuto** [comment]
Your changes showed up.

I have two more small comments (above) and then I think this should be good to go :)

**Comment by mterwill** [comment]
Looks like all of @ppannuto and my suggestions are implemented – merging.

**Pull request merged**